src/app/home/pages/PlansPage/helpers.ts

```diff
@@ -74,10 +74,29 @@ export const getPlanInfo = (plan: IPlan) => {
   const latestMigAnalytic = plan.Analytics ? plan.Analytics[0] : null;
   const isMaxResourcesLimitReached =
     latestMigAnalytic?.status?.analytics?.k8sResourceTotal > 10000 ? true : false;
+  let migrationType =
+    plan?.MigPlan?.metadata?.annotations['migration.openshift.io/selected-migplan-type'];
+  if (migrationType === undefined) {
+    if (plan.MigPlan.status?.conditions) {
+      plan.MigPlan?.status?.conditions?.forEach((element) => {
+        if (element.type === 'MigrationTypeIdentified') {
```

Comment:
It might be nice to factor out this switch logic into a shared helper function somewhere, but that's not critical.